### ISSUE TITLE
Reproducing RotMNIST results

### ISSUE THREAD

**Issue by AllanYangZhou** [g-benton/learning-invariances#1]
Hi,

Thank you for releasing this code, and I quite liked the paper.

I've had some difficulty getting the RotMNIST experiments working. From `./experiments/rotmnist` I'm simply running `python runner.py` with the default argparse setting. I've slightly modified this file only to do a train/val split and print the train/val accuracies (exact code [here](https://github.com/g-benton/learning-invariances/compare/master...AllanYangZhou:master)).

I'm getting poor training and val accuracies that are not improving after the first few epochs (see below)--the numbers are below the same architecture trained naively (without any augmentation). Should I be using different hparams from the default? Or does it just require more epochs? 

Any advice or tips would be greatly appreciated!

Thanks!

<img width="1253" alt="Screen Shot 2020-12-17 at 4 16 58 PM" src="https://user-images.githubusercontent.com/8732147/102544775-4c4de980-4083-11eb-95f9-f3c81f8e80ad.png">

Other info: python 3.7.8, torch 1.7.0, cuda 10.1. System is Ubuntu 16.04.

**Comment by g-benton** [comment]
Hi Allan,

Thanks for the heads up - for me it looks the primary issue is that the default learning rate is just too high. If I drop the learning rate to 0.01 I am getting training accuracy of 0.85-0.9 by about the third epoch. 

The network in that file is also a very small network just to keep things easy to run, if you want to get better accuracy you can swap the model definition line in the `runner.py` file with:

net = models.layer13s(in_channels=1,num_targets=10, k=32)

(You can also push `k` up to 128, but it will take a bit longer to train). For me the layer13s gave >90% train accuracy after 3 epochs.

Let me know if that helps!

Best,
Greg

**Comment by AllanYangZhou** [comment]
Yes, adjusting the learning rate did the trick. I probably should've tried that earlier--I'll also try increasing the architecture size. Thank you for the prompt reply!

Best,
Allan
